src/modules/Builder/components/Configuration/components/Main/Main.jsx

```diff
@@ -106,9 +116,10 @@ export default class Main extends Component {
                         />
                     </div>
                     <div className="form-group">
-                        <Select
+                        <SelectSearch
+                            id="timeZones"
                             key="time-zone"
-                            label="Time zone"
+                            label="Time zones"
```

Comment:
I think that the word should be singular here, since you can only select one time zone.